pkg/util/filesystem/util_unix.go

```diff
@@ -35,3 +36,6 @@ func IsUnixDomainSocket(filePath string) (bool, error) {
 	}
 	return true, nil
 }
+
+// IsAbs is filepath.IsAbs.
+var IsAbs = filepath.IsAbs
```

Comment:
nit, slight preference for `func IsAbs(path string) bool { return filepath.IsAbs(path) }` to avoid mutable package var

Reply:
Addressed